vellum/workflows/triage_message/nodes/read_message_node.py

```diff
@@ -73,10 +73,15 @@ def run(self) -> Outputs:
                     )
                 )
                 session.commit()
+                contact_source = "Unknown"
+                if hasattr(inbox_message, 'contact') and inbox_message.contact:
+                    contact = inbox_message.contact
+                    contact_source = contact.slack_display_name or contact.full_name or contact.email or "Unknown"
+                
                 message = SlimMessage(
                     message_id=inbox_message.id,
                     body=inbox_message.body,
-                    source=inbox_message.source,
+                    source=contact_source,
```

Comment:
let's have the contact's email, id, fullname, and slack display as top level to this `SlimMessage`, update throughout the workflow